extensions/amp-story/1.0/amp-story-access.css

```diff
@@ -110,9 +109,17 @@ amp-story-access.i-amphtml-story-access-visible::before {
     flex-direction: column !important;
     margin: 0 !important;
     max-height: 80vh !important;
```

Comment:
nit: can you check with @hongwei1990 to see if this should be smaller?

Reply:
Right, she'd like us to use 60, done!